src/routes/auth/local.js

```diff
@@ -2,11 +2,11 @@ import express from 'express';
 import AuthLocalController from '../../controllers/AuthLocalController';
 import asyncHandler from '../../middlewares/asyncHandler';
 import verifyToken from '../../middlewares/verifyToken';
-// import validateUser from '../../middlewares/validateUser';
+import validateUser from '../../middlewares/validateUser';
 
 const router = express.Router();
 
-router.post('/signup', AuthLocalController.signup);
+router.post('/signup', validateUser, AuthLocalController.signup);
```

Comment:
how are you error handling this post?

Reply:
Hello @e-liyai. I worked on signing up a new user. 
Previously in `develop`,  we did not have an error handler. For now, I introduced an error handling algorithm in #34 which is not yet merged